client/src/components/TestSuite/TestSuiteDetails/TestListItem/TestListItem.tsx

```diff
@@ -99,15 +100,23 @@ const TestListItem: FC<TestListItemProps> = ({
     </ReactMarkdown>
   );
 
+  const getResultIcon = () => {
+    if (testRunInProgress && test.isInCurrentTestRun) {
+      return <PendingIcon color="disabled" />;
+    } else {
+      if (test.result) {
+        return <ResultIcon result={test.result} />;
+      } else {
+        return <></>;
+      }
+    }
+  };
+
   return (
     <>
       <Box className={styles.listItem}>
         <ListItem>
-          {test.result && (
-            <div className={styles.testIcon}>
-              <ResultIcon result={test.result} />
-            </div>
-          )}
+          <div className={styles.testIcon}>{getResultIcon()}</div>
```

Comment:
Delete the padding from the styles file for `testIcon`

Reply:
![image](https://user-images.githubusercontent.com/45436454/155385900-643506b5-c333-4125-b719-93fcb878fef8.png)
Doesn't this then make it too squished?

Reply:
Hm, wasn't doing that for me. I'll take another look.

Reply:
Ok yeah my UI is missing the test ID for some reason, which was causing unnecessary padding to show up. This can stay in, I'll investigate the issue elsewhere.